src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SqlNormalizer.cs

```diff
@@ -129,12 +136,12 @@ private object DeNormalizeInternal(Type t, Stream s)
         {
             object result = null;
             //if nullable and not the top object, read the null marker
-            if (!_isTopLevelUdt && typeof(INullable).IsAssignableFrom(t))
+            if (!_isTopLevelUdt && typeof(System.Data.SqlTypes.INullable).IsAssignableFrom(t))
```

Comment:
There are two more usage for `INullable` interface in this file; If you like to use the whole name of it like this, please apply the same method for all of them and remove the using namespace from the top of file.

Reply:
The two files were a mix of styles. Which do you prefer?

Reply:
I'd say `if (!_isTopLevelUdt && typeof(INullable).IsAssignableFrom(t))` to keep it identical like the other usages.

Reply:
Done.